vscode/src/toitExec.ts

```diff
@@ -69,8 +66,8 @@ async function executeDeployCommand(ctx: Context, device?: Device) {
   try {
     if (!device) device = await selectDevice(ctx, { "activeOnly": false, "simulatorOnly": false });
 
-    const { stdout, stderr } = await execFile(ctx.toitExec, [ "dev", "-d", device.name, "deploy", filePath ]);
-    ctx.output(device.deviceID, device.name, stdout, stderr);
+    ctx.startDeviceOutput(device);
+    await execFile(ctx.toitExec, [ "dev", "-d", device.name, "deploy", filePath ]);
```

Comment:
Same: where is the output of the command piped into the output?

Edit: it seems like we are falling back to `logs`.